src/generic_ui/polymer/settings.ts

```diff
@@ -46,7 +33,7 @@ Polymer({
     this.$.confirmNewServer.hidden = false;
   },
   resetStunServers: function() {
-    browserified_exports.model.globalSettings.stunServers = this.DEFAULT_STUN_SERVERS;
+    browserified_exports.model.globalSettings.stunServers = [];
```

Comment:
Sending the servers as an empty array resets them to the default. (see globals.ts:51)